test/snippet/io/structure_file/structure_file_output_pass_rec.cpp

```diff
@@ -0,0 +1,29 @@
+#include <sstream>
+
+#include <seqan3/io/structure_file/output.hpp>
+#include <seqan3/io/structure_file/input.hpp>
+
+auto input = R"(> S.cerevisiae_tRNA-PHE M10740/1-73
+GCGGAUUUAGCUCAGUUGGGAGAGCGCCAGACUGAAGAUUUGGAGGUCCUGUGUUCGAUCCACAGAAUUCGCA
+(((((((..((((........)))).((((.........)))).....(((((.......)))))))))))). (-17.50)
+> example
+UUGGAGUACACAACCUGUACACUCUUUC
+..(((((..(((...)))..)))))... (-3.71))";
+
+int main()
+{
+    bool criteria = true;
+    seqan3::structure_file_input fin{std::istringstream{input},
+                                     seqan3::format_vienna{},
+                                     seqan3::fields<seqan3::field::ID, seqan3::field::SEQ, seqan3::field::STRUCTURE>{}};
+    // the output doesn't have to match the configuration of the input
+    seqan3::structure_file_output fout{std::ostringstream{}, seqan3::format_vienna{}};
+
+    for (auto & r : fin)
+    {
+        if (criteria) // r fulfills some filter criterium
+        {
```

Comment:
the braces could be removed

Reply:
but again what is this snippet trying to explain?

Reply:
I guess something about passing records to the output object?